cmd/queue/main.go

```diff
@@ -360,17 +378,7 @@ func main() {
 		zap.String(logkey.Pod, env.ServingPod))
 
 	if err := validateEnv(env); err != nil {
-		logger.Fatal(err.Error())
-	}
-
-	// If this is set, we run as a standalone binary to probe the queue-proxy.
-	if *readinessProbeTimeout >= 0 {
-		if err := probeQueueHealthPath(logger, env.QueueServingPort, *readinessProbeTimeout, env); err != nil {
-			// used instead of the logger to produce a concise event message
-			fmt.Fprintln(os.Stderr, err)
-			os.Exit(1)
-		}
-		os.Exit(0)
+		log.Fatalf(err.Error())
```

Comment:
Why change the logger here? The "correct" logger is actually already built for you.

I think we want to move this up to right after parsing the environment to validate the `env` before using it in the `probeQueueHealthPath` function.

Reply:
@markusthoemmes Hmm I see. I might have misunderstood your comment here https://github.com/knative/serving/issues/6808#issuecomment-584700705 where exactly do we want to use `log.Fatalf(...)`?